Source/MultipartFormData.swift

```diff
@@ -256,16 +256,17 @@ public class MultipartFormData {
 
         var isReachable = false
 
-        do {
-            try fileURL.checkResourceIsReachable()
+        if #available(OSX 10.10, *) {
```

Comment:
Just to clear this up, Alamofire 2.0 is not going to support iOS 7. The README in the `swift-2.0` branch reflects that decision.